### ISSUE TITLE
[Bug Report] Expansion panel content doesn't set width to 100%

### ISSUE THREAD

**Issue by rbanks54** [vuetifyjs/vuetify#9828]
### Environment
**Vuetify Version:** 2.1.12
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.108
**OS:** Windows 10

### Steps to reproduce
In the codepen, expand one of the panels and you'll see the content is set left.
Adjust the template to set the v-expansion-panel-content style to have width:100% and the content appears as expected.

This occurs because of the v-row on line 8 of the codepen (it works if I use v-container)

### Expected Behavior
expansion panel content should fill the width of the parent element

### Actual Behavior
expansion panel content is not using full width

### Reproduction Link
<a href="https://codepen.io/rbanks54/pen/qBEWWOq" target="_blank">https://codepen.io/rbanks54/pen/qBEWWOq</a>

**Comment by jacekkarczmarczyk** [comment]
What do you need v-row for? It works fine without it https://codepen.io/jkarczm/pen/gObzXdE?editors=1010
I don't think we want support markup when the expansion panel header/content is not a direct child of expansion panel. @vuetifyjs/core-team ?

**Comment by johnleider** [comment]
Just use the **grow** helper class: https://codepen.io/johnjleider/pen/abzKYmN?editors=1010

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by rbanks54** [comment]
> What do you need v-row for? It works fine without it https://codepen.io/jkarczm/pen/gObzXdE?editors=1010
> I don't think we want support markup when the expansion panel header/content is not a direct child of expansion panel. @vuetifyjs/core-team ?

The expansion panel contents are in a separate component and given SFC templates need a single root element, I used a v-row as a simple wrapper. I could have used something else I suppose (such as v-container) but I chose row at the time.

@johnleider  Thanks for note about the grow helper class. It's one of those things I hadn't paid much attention to when running through v2.0 docs. Thanks again!
